chaos_genius/core/utils/data_loader.py

```diff
@@ -23,6 +21,7 @@
 
 
 class DataLoader:
+    """Data Loader Class."""
```

Comment:
This provides no value over not having a docstring. Needs to be a bit more descriptive and ideally should also mention any considerations/caveats before using this class.